src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_backup_commands.py

```diff
@@ -3,17 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
-import time
 import unittest
-
-LOCATION = "eastus2euap"
+LOCATION = "southcentralusstage"
 
 
 @unittest.skip("Backup unstable so these tests need to be skipped for this version")
 class AzureNetAppFilesBackupServiceScenarioTest(ScenarioTest):
     def setup_vnet(self, vnet_name, subnet_name):
         self.cmd("az network vnet create -n %s -g {rg} -l %s --address-prefix 10.5.0.0/16" %
-                 (vnet_name, LOCATION))
+                 (vnet_name, "southcentralus"))
```

Comment:
Why do you use `southcentralus` instead of `LOCATION`?

Reply:
@jsntcy That is because CBS (Cloud Backup Service) is only properly set up in 2 test regions, eastus2euap and southcentralusstage. There has been some problems with Eastus2euap so I had to switch to southcentralusstage. In southcentralusstage I can only perform netappfiles operations so to be able to create a vnet and subnet for the volume it has to be created in southcentralus.